lib/cifar10.ex

```diff
@@ -36,6 +40,9 @@ defmodule Scidata.CIFAR10 do
     * `:transform_labels` - similar to `:transform_images` but applied to
       dataset labels
 
+    * `:test_set` - indicate whether the training set or the test set
```

Comment:
Should we use an option or a separate function? 🤔

Reply:
We could go with a separate function! Like `download_test/1`?

Reply:
I am unsure. Given they return different results (but still a tuple), I like a different function. /cc @wojtekmach @seanmor5

Reply:
I think a separate function makes sense and just pushed related changes. I went with `download_test/1` but maybe `download_test_set/1` is more clear. `download_train/1` instead of `download/1` might be better too...

Reply:
FWIW, `tfds.load` has a `split` option where you specify a list of the data splits you want: `tfds.load("mnist", split=['train', 'test', 'validation']`...

But I think separate functions as a separation of responsibilities is good as well